uk.co.compucorp.civicrm.hrcontactactionsmenu/templates/CRM/HRContactActionsMenu/Page/Inline/Actions.tpl

```diff
@@ -1,69 +1,70 @@
 <div id="crm-contact-actions-wrapper" data-edit-params='{ldelim}"cid": "{$contactId}", "class_name": "CRM_Contact_Page_Inline_Actions"{rdelim}'>
   {crmButton id="crm-contact-actions-link" href="#" icon="bars"}
-  {ts}Actions{/ts}
+    {ts}Actions{/ts}
   {/crmButton}
-  <div id="crm-contact-actions-list">
+  <div id="crm-contact-actions-list" class="crm_contact-actions">
     <div id="bootstrap-theme">
-      <div class="row">
-        <div class="col-md-4 crm_contact_action_menu__user_info">
-            {foreach from=$menu->getHighlightedPanelItems() item='group'}
-            <div class="row">
+      <div class="crm_contact-actions__body">
+        <div class="col-md-4 crm_contact-actions__body__user-info crm_contact-actions__user-info--margin">
+          {foreach from=$menu->getHighlightedPanelItems() item='group'}
+            <div class="col-md-12">
               <h3>{$group->getTitle()}</h3>
               {foreach from=$group->getItems() item='item'}
                 {$item->render()}
               {/foreach}
             </div>
-            {/foreach}
+          {/foreach}
+        </div>
+        <div class="col-md-8 crm_contact-actions__body__user-actions crm_contact-actions--bg-white">
```

Comment:
what's this `crm_contact-actions--bg-white` class for? Please add screenshot

Reply:
<img width="1503" alt="screen shot 2018-01-24 at 1 44 22 pm" src="https://user-images.githubusercontent.com/6307362/35320679-bfbef738-010c-11e8-81d6-3b2f4bdcfa5e.png">

This class adds white background to user-actions container as above.

Reply:
What if the left panel is going to be gray in the future? Are you going to change the class name as well, or end up with a class name (white) that is for a different color (gray)?

If you leverage BEM, you can have `contact-actions`, `contact-actions__primary-panel`, `contact-actions__secondary-panel` (i'm using "primary" and "secondary" instead of "left" and "right" because you don't know how the panel might change in the future)

And then you can do

```scss
.contact-actions__primary-panel {
  color: xyz;
}

.contact-actions__secondary-panel {
  color: zyx;
}
```